gmime/gmime_test.go

```diff
@@ -671,6 +672,103 @@ func TestPart_String(t *testing.T) {
 	}
 }
 
+func TestPart_GetHeader(t *testing.T) {
+	type TestCase struct {
+		filename string
+		expected []map[string]string
+	}
+	tcs := []TestCase{
+		{
+			filename: `test_data/mime_simple.eml`,
+			expected: []map[string]string{
+				{
+					"Content-Type": "text/html",
+				},
+				{
+					"Content-Transfer-Encoding": "quoted-printable",
```

Comment:
can we have a test payload with the infamous `=3D` line delimiters?

Reply:
That isn't going to get parsed here, this is only to extract the headers but not to decode quoted printable